types/styled-components/native.d.ts

```diff
@@ -206,14 +206,6 @@ export interface ReactNativeStyledInterface<T extends object> extends ReactNativ
     typeof ReactNative.ViewPagerAndroid,
     T
   >;
-  FlatList: ReactNativeThemedStyledFunction<
```

Comment:
this change is suspicious. I don't know styled-components, but this might have incorrectly only been added to the <=3.6 types since the >=3.7 types were in a subdirectory.

Reply:
I'm going to add it, #42092 definitely just changed the old version and not the newer one.